.github/workflows/release-dev.yml

```diff
@@ -35,6 +35,7 @@ jobs:
           jq --arg newVersion "$devVersion" '.version = $newVersion' package.json > package.tmp && mv package.tmp package.json
           jq --arg newVersion "$devVersion" '.version = $newVersion' package-lock.json > package-lock.tmp && mv package-lock.tmp package-lock.json
           jq --arg newVersion "$devVersion" '.packages[""].version = $newVersion' package-lock.json > package-lock.tmp && mv package-lock.tmp package-lock.json
+          jq --null-input --arg version "$devVersion" '{"name": "@pinecone-database/pinecone", "version": $version}' > src/version.json
```

Comment:
For dev builds, we never relied on the `npm version` command to set version numbers. So we update `src/version.json` a little differently also. I tested this jq command on my local machine.